_posts/minecraft/bedrock/2024-07-16-server-properties.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: post
+title:  "Configuring Minecraft Server: Server Properties"
+category: 
+    - Bedrock
+    - Getting-started
+tags: Server
+description: "Explore the commonly used attributes of the server.properties file to effectively configure your Bedrock Minecraft server"
+permalink: /minecraft/bedrock/server/server-properties
+toc: false
+---
+
+## Introduction
+
+Understanding the server.properties file is crucial for configuring and customizing your Minecraft server. This guide highlights the essential attributes within the file, enabling you to optimize your server settings for the best gameplay experience.
+
+## server.properties
+
+### server-name
+
+Possible Values: String  
+Default Value: Dedicated Server  
+
+Specifies the name of the server as it appears in the multiplayer server list.
+
+- Helps players identify and connect to the server easily.
+
+---
+
+### gamemode
+
+Possible Values: survival, creative, adventure  
+Default Value: survival  
+
+Defines the mode of gameplay.
+
+- If set to `survival`, players must gather resources, build structures, and survive against hostile mobs.
+- If set to `creative`, players have unlimited resources, can fly, and are not attacked by hostile mobs.
+- If set to `adventure`, players can interact with objects like levers and buttons but cannot break blocks.
```

Comment:
I didnt mean to literally explain the most well known and obvious stuff 😭

Reply:
😭